Patches/Dragon's Descent/ThingDefs_Races/Patch_Races_Animal_Common_Dragon.xml

```diff
@@ -0,0 +1,1121 @@
+							<cooldownTime>3.5</cooldownTime>
+							<power>86</power>
+							<capacities>
+								<li>Scratch</li>
+							</capacities>
+							<linkedBodyPartsGroup>FrontRightPaw</linkedBodyPartsGroup>
+							<label>right claw</label>
+							<armorPenetrationSharp>24.09</armorPenetrationSharp>
+							<armorPenetrationBlunt>120.417</armorPenetrationBlunt>
+							<chanceFactor>0.25</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>4.41</cooldownTime>
+							<power>92</power>
+							<capacities>
+								<li>Bite</li>
+							</capacities>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<label>teeth</label>
+							<armorPenetrationSharp>4.75</armorPenetrationSharp>
+							<armorPenetrationBlunt>301.042</armorPenetrationBlunt>
+							<chanceFactor>0.2</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>2.94</cooldownTime>
+							<power>131</power>
+							<capacities>
+								<li>Rend</li>
+							</capacities>
+							<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+							<label>teeth</label>
+							<armorPenetrationSharp>10.67</armorPenetrationSharp>
+							<armorPenetrationBlunt>677.344</armorPenetrationBlunt>
+							<chanceFactor>0.15</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>11.12</cooldownTime>
+							<power>69</power>
+							<capacities>
+								<li>Cut</li>
+							</capacities>
+							<surpriseAttack>
+							  <extraMeleeDamages>
+								<li>
+								  <def>Stun</def>
+								  <amount>30</amount>
+								</li>
+							  </extraMeleeDamages>
+							</surpriseAttack>
+							<linkedBodyPartsGroup>Horns</linkedBodyPartsGroup>
+							<label>horn</label>
+							<armorPenetrationSharp>0.84</armorPenetrationSharp>
+							<armorPenetrationBlunt>16.725</armorPenetrationBlunt>
+							<chanceFactor>0.05</chanceFactor>
+						</li>
+						<li Class="CombatExtended.ToolCE">
+							<cooldownTime>11.12</cooldownTime>
+							<power>38</power>
+							<capacities>
+								<li>Stab</li>
+							</capacities>
+							<surpriseAttack>
+							  <extraMeleeDamages>
+								<li>
+								  <def>Stun</def>
+								  <amount>30</amount>
+								</li>
+							  </extraMeleeDamages>
+							</surpriseAttack>
+							<linkedBodyPartsGroup>Horns</linkedBodyPartsGroup>
+							<label>horn</label>
+							<armorPenetrationSharp>1.68</armorPenetrationSharp>
+							<armorPenetrationBlunt>16.725</armorPenetrationBlunt>
+							<chanceFactor>0.1</chanceFactor>
+						</li>
+					</tools>
+				</value>
+			</li>
+			<!-- Miscellaneous -->
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="Black_Dragon"]/race/wildness</xpath>
+				<value>
+					<wildness>0.99</wildness>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/PawnKindDef[defName="Black_Dragon"]/combatPower</xpath>
+				<value>
+					<combatPower>2190</combatPower>
```

Comment:
Out of curiosity, how'd you come by this value?

Reply:
After checking out the values for the thrumbo (vanilla and patched), I'll be removing these patch operations.